examples/cpp/pyperf/PyPerfNativeStackTrace.h

```diff
@@ -8,18 +8,27 @@
 
 #include <string>
 #include <vector>
+#include <map>
 
 namespace ebpf {
 namespace pyperf {
 
+typedef struct {
+  unw_cursor_t cursor;
+  unw_addr_space_t as;
+  void *upt;
+  time_t timestamp;
+} Object;
```

Comment:
Let's give it a more explicit name? `UnwindCacheEntry`?

Reply:
Renamed